en/django_installation/instructions.md

```diff
@@ -172,15 +172,29 @@ Before we do that, we should make sure we have the latest version of `pip`, the
 A requirements file keep a list of dependencies to be installed using
 `pip install`:
 
-First, run `pip install django~=2.0.6` to install Django.
+First create a `requirements.txt` file inside of `djangogirls/` folder:
+
+```
+djangogirls
+└───requirements.txt
+```
+
+In your `djangogirls/requirements.txt` file you should add the following text:
+
+{% filename %}djangogirls/requirements.txt{% endfilename %}
+```
+Django==2.0.6
+```
+
+Now, run ``pip install -r requirements.txt`` to install Django.
 
 {% filename %}command-line{% endfilename %}
 ```
-(myvenv) ~$ pip install django~=2.0.6
-Collecting django
-  Downloading Django-2.0.6-py2.py3-none-any.whl (6.8MB)
-Installing collected packages: django
-Successfully installed django-2.0.6
+(myvenv) ~$ pip install -r requirements.txt
+Collecting Django==2.0.6 (from -r requirements.txt (line 1))
+  Downloading Django-2.0.6-py3-none-any.whl (7.1MB)
+Installing collected packages: Django
+Successfully installed Django-2.0.6
```

Comment:
Maybe having the `Django==2.0.6` entries as `Django==X.Y.Z` would make this more portable moving forward?

Reply:
hm? sorry @goanpeca I'm confused about this change, your are suggesting about making the django version more flexible for future modifications?

Reply:
I am saying that the output of installing django (via pip with specification or requirements) might change due to versions of pip or other things so maybe having something like:

```
Collecting Django==X.Y.Z (from -r requirements.txt (line 1))
  Downloading Django-X.Y.Z-py3-none-any.whl (7.1MB)
Installing collected packages: Django
Successfully installed Django-X.Y.Z
```

is better, simpler? and avoids having to update version numbers in different places (which is error prone?) OR is there a way to define that version in a single place and the use as a variable replacement (like a jinja variable?)

```jinja2
{% set django_version = '2.0.6' %}
{{ django-version}} 
```

(I am not sure how this is rendered :-p so this might make no sense?)
```
Collecting Django=={{ django-version}}  (from -r requirements.txt (line 1))
  Downloading Django-{{ django-version}} -py3-none-any.whl (7.1MB)
Installing collected packages: Django
Successfully installed Django-{{ django-version}} 
```

Reply:
@goanpeca I put ``django_version`` as a variable in ``book.json``, can you review again please?

Reply:
Looks great :-)